src/JSONSettingsRESTAPI/Endpoints/SettingsGetEndpoint.cpp

```diff
@@ -39,7 +40,7 @@ namespace systelab { namespace setting { namespace rest_api {
 			}
```

Comment:
namespace "systelab": Did you know if someone outside the company, uses any of this llibraries?

Reply:
Not aware of it, but it could be possible.
However, the use of this namespace was agreed between the different c++ projects to distinguish what is shared and what belongs to each project.